proto/evmos/fees/v1/genesis.proto

```diff
@@ -32,4 +32,9 @@ message Params {
   ];
   // parameter to configure the cost of address derivation
   uint64 addr_derivation_cost_create = 4;
+  // parameter to configure the minimum gas price value for transactions
```

Comment:
try to keep the parameter name add the beginning of the comment:

```suggestion
  // min_gas_price defines the minimum gas price value for eth transactions
```

Reply:
I will amend. With the note that `min_gas_price` is for both cosmos and eth transactions.